notebooks/official/matching_engine/similar_products_recommender_using_matching_engine.ipynb

```diff
@@ -0,0 +1,1683 @@
+{
```

Comment:
<p>Create the indexes</p><p> </p><p>A ANN  -&gt; An ANN</p><p> </p><h3>Create ANN Index (for Production Usage) -&gt; Create an ANN index for production usage</h3>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>done</p>